plasma_framework/docs/audits/README.md

```diff
@@ -0,0 +1,9 @@
+## Audits 
+
+The following list contains information on all the audits that have been conducted for the Plasma contracts component of the OMG network. 
+
+| Release | Organization | Period | Report |
+|--|--|--|--|
+| [v0.0.1](https://github.com/omisego/plasma-contracts/releases/tag/v0.0.1) | Quantstamp | 24.09.2018 - 10.11.2018 | [Report](https://github.com/omisego/plasma-contracts/blob/v0.0.1/docs/quantstamp-audit-3cc6097.md)|
+| [v1.0.0](https://github.com/omisego/plasma-contracts/releases/tag/v1.0.0) | Consensys Diligence| 21.10.2019 - 22.11.2019 and 16.12.2019 - 03.01.2020 | [Report](https://diligence.consensys.net/audits/2020/01/omisego-morevp/) |
```

Comment:
have we considered to have a copy of pdf ourselves? I don't know, probably we can have link to pdf under this repo + the link to their page?